File: pkg/codegen/testing/test/testdata/multiline-string-pp/nodejs/multiline-string.ts
import * as pulumi from "@pulumi/pulumi";
import * as random from "@pulumi/random";

const foo = new random.RandomShuffle("foo", {inputs: [
    `just one
newline`,
    `foo
bar
baz
qux
quux
qux`,
    `{
    "a": 1,
    "b": 2,
    "c": [
      "foo",
      "bar",
      "baz",
      "qux",
      "quux"
    ]
}
`,
]});

File: pkg/codegen/testing/test/testdata/multiline-string-pp/python/multiline-string.py
import pulumi
import pulumi_random as random

foo = random.RandomShuffle("foo", inputs=[
    """just one
newline""",
    """foo
bar
baz
qux
quux
qux""",
    """{
    "a": 1,
    "b": 2,
    "c": [
      "foo",
      "bar",
      "baz",
      "qux",
      "quux"
    ]
}
""",
])
